net/foolsm/Makefile

```diff
@@ -0,0 +1,62 @@
+#
+# Copyright (C) 2010-2011 OpenWrt.org
```

Comment:
no.  either drop this outright, or use your own name here.

Reply:
Doesn't seem right to just remove somebody else's copyright statement.

Why is this necessary to having this package added (back) to OpenWrt?

Reply:
Because it's not copyright 2010-2011 openwrt.  It's a new file with your own work in it, and I don't believe you did it in 2010, nor is your name openwrt.org

Reply:
It's not a new file with my own work in it.  It's being "resurrected" [from OpenWRT history](https://dev.openwrt.org/browser/packages/net/lsm/Makefile).